apidef/oas/root.go

```diff
@@ -76,6 +76,8 @@ type Info struct {
 	State State `bson:"state" json:"state"` // required
 	// Versioning holds configuration for API versioning.
 	Versioning *Versioning `bson:"versioning,omitempty" json:"versioning,omitempty"`
+	// ClassicAPIID is the ID of the classic API definition this OAS api is staged from during migration.
+	ClassicAPIID string `bson:"classicApiId,omitempty" json:"-"`
```

Comment:
```suggestion
	ClassicAPIID string `bson:"classicApiId,omitempty" json:"classicApiId,omitempty"`
```
We cannot skip the json marshalling here as we're not directly using `bson` to store to db. 
The oas object is serialized to json and stored as a blob in db object